lessons/00-getting-ready/README.md

```diff
@@ -152,3 +149,20 @@ node_modules
 ```
 
 Which will ignore the `node_modules` folder — standard for a Node repository.
+
+##Class Overview
+
+###What will class look like?
+We'll spend the first ~five weeks of class on a series of topics which are central to web development, including JavaScript, Node and Express, databases, jQuery and Ajax, templating and CSS, APIs, clientside frameworks, and deployment and scaling.
+
+For the three or four weeks after that, you'll work on labs in pairs, then you'll spend the rest of the semester on a team project.
+
+For this semester we are trying out a couple different techniques in class:
+
+* **Lectures** will never be more than 30 minutes long, and will provide an overview of the lesson for the day.
+* **Individual Readings** in the form of READMEs will be done in class at your own pace.  You are encouraged to ask the teaching team any questions and discuss things with your classmates! The homework assignment for the next class will be at the end of these READMEs.  You are welcome to skip around the reading however you like, or start your homework in class.
+* **Read-shares** are a think-pair-share-inspired way to help us break down complex concepts. We'll assign small things for each student to research for ~5 minutes, then we'll go around the room and everyone will share what they learned (with the teaching team filling in the blanks).
+
+## Before Class 1 (Tuesday 1/17/17)
+- Follow the instructions above to set up your development environment. If something is giving you problems we can help once class starts, but try to get as much done as you can before class.
+- Complete at least the first 10 levels (up through move2) of [this Git tutorial](http://pcottle.github.io/learnGitBranching). (Optional: If you want a better high-level understanding of git pushing and pulling, I recommend [this video](https://www.youtube.com/watch?v=8KCQe9Pm1kg). If you want a better code-level understanding, I also recommend [this video](https://www.youtube.com/watch?v=Y9XZQO1n_7c).)
```

Comment:
Should we also get them to submit a "preclass" survey on things that they didn't understand with git, just so they start getting in the habit of filling out the forms with every reading?